### ISSUE TITLE
Eval results 

### ISSUE THREAD

**Issue by 1664403370** [google-research/bert#699]
Why is my running result like this? Is the parameter not passed in? what should I do? Thank you

INFO:tensorflow:***** Eval results *****
INFO:tensorflow:  global_step = 9
INFO:tensorflow:  loss = 0.0
INFO:tensorflow:  masked_lm_accuracy = 0.0
INFO:tensorflow:  masked_lm_loss = 0.0
INFO:tensorflow:  next_sentence_accuracy = 0.0
INFO:tensorflow:  next_sentence_loss = 0.0
INFO:tensorflow:  next_sentence_accuracy = 0.0
INFO:tensorflow:  next_sentence_loss = 0.0

**Comment by guotong1988** [comment]
Same question.
